submissions/VolodymyrRutskyi/tiny-js-world/index.js

```diff
@@ -1,90 +1,52 @@
-/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
-   Complete the below for code reviewers' convenience:
+class Inhabitant {
+   constructor(species, name, gender, legs, saying){
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying;
+      this.props = ['species', 'name', 'gender', 'legs', 'saying']
+   }
+   showProps() {
+      return this.props.map(prop => this[prop]).join('; ');
+   }
+}
+
+class Human extends Inhabitant{
+   constructor (species, name, gender, saying){
```

Comment:
Are you sure a `Human` needs a `species` here? A human can only be human, isn't it?